people/qiaohai.py

```diff
@@ -0,0 +1,5 @@
+import numpy.matlib 
+import numpy as np 
+
+# Generate a diagonal matrix with 3 rows and 4 columns.  
+print (np.matlib.eye(n =  3, M =  4, k =  0, dtype =  float))
```

Comment:
code style suggestions: please refer to PEP-8 coding style and update this line of code. (basically, we just need one space before and after "=".